tests/forking_agent/test_agent_executor.py

```diff
@@ -361,17 +362,21 @@ def get_modification_datetime(file: str) -> datetime.datetime:
         return datetime.datetime.fromtimestamp(os.stat(file).st_mtime)
 
     old_check_executor1 = get_modification_datetime(f"{executor_1.executor_virtual_env.env_path}/.inmanta_env_status")
+    old_check_executor2 = get_modification_datetime(f"{executor_2.executor_virtual_env.env_path}/.inmanta_env_status")
 
-    executor_1.check_modification_time_venv()
-    executor_2.check_modification_time_venv()
+    time.sleep(2)
```

Comment:
1. async sleep is more appropriate 
2. we can probably take even shorter timings?